src/OpenTelemetry.Exporter.Zipkin/OpenTelemetry.Exporter.Zipkin.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
```

Comment:
I think we should add net461 as well here. That way .NET Framework 4.6.1+ targets will get System.Text.Json instead of Newtonsoft.Json.

Reply:
Ok! thank you!